src/azure-cli/azure/cli/command_modules/appconfig/_azconfig/models.py

```diff
@@ -81,7 +81,7 @@ class QueryKeyValueOptions(object):
         An ID that can be used to correlate the request with a more general operation.
     '''
 
-    empty_label = u'\0'
+    empty_label = '\\0'
```

Comment:
Why do you change empty label to string value '\0' here?

Reply:
Explained under another comment. 

---
In reply to: [347079825](https://github.com/Azure/azure-cli/pull/11036#discussion_r347079825) [](ancestors = 347079825)